docs/standard/exceptions/best-practices-for-exceptions.md

```diff
@@ -37,31 +38,31 @@ If you don't check connection state before closing, you can catch the `InvalidOp
 
 The method to choose depends on how often you expect the event to occur.
 
-- Use exception handling if the event doesn't occur very often, that is, if the event is truly exceptional and indicates an error (such as an unexpected end-of-file). When you use exception handling, less code is executed in normal conditions.
+- Use exception handling if the event doesn't occur often, that is, if the event is truly exceptional and indicates an error such as an unexpected end-of-file. When you use exception handling, less code is executed in normal conditions.
 
 - Check for error conditions in code if the event happens routinely and could be considered part of normal execution. When you check for common error conditions, less code is executed because you avoid exceptions.
 
 ## Design classes so that exceptions can be avoided
 
-A class can provide methods or properties that enable you to avoid making a call that would trigger an exception. For example, a <xref:System.IO.FileStream> class provides methods that help determine whether the end of the file has been reached. These can be used to avoid the exception that is thrown if you read past the end of the file. The following example shows how to read to the end of a file without triggering an exception.
+A class can provide methods or properties that enable you to avoid making a call that would trigger an exception. For example, a <xref:System.IO.FileStream> class provides methods that help determine whether the end of the file has been reached. This kind of approach can be used to avoid the exception that is thrown if you read past the end of the file. The following example shows how to read to the end of a file without triggering an exception.
```

Comment:
This kind of approach >> Maybe we can just say "This approach"

Reply:
Ok, updated the content.